File: rb/utility/make_context.rb
# IndoorEnvironmentalMonitoring SDK utility: make_context
require_relative '../core/context'
module IndoorEnvironmentalMonitoringUtilities
  MakeContext = ->(ctxmap, basectx) {
    IndoorEnvironmentalMonitoringContext.new(ctxmap, basectx)
  }
end

File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// IndoorEnvironmentalMonitoring SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class IndoorEnvironmentalMonitoringMakeContext
{
    public static function call(array $ctxmap, ?IndoorEnvironmentalMonitoringContext $basectx): IndoorEnvironmentalMonitoringContext
    {
        return new IndoorEnvironmentalMonitoringContext($ctxmap, $basectx);
    }
}
